CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @operator-framework/rukpak-maintainers
```

Comment:
Does this prohibit those from outside the operator-framework org to become approvers?

Reply:
Seems like this is not an issue.

Reply:
It's not super clear reading from https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners but I think this is fine for now until we have an explicit need to have someone listed as a maintainer that's not also in the o-f organization.